frontend/portals/radiant/tailwind.config.js

```diff
@@ -6,7 +6,10 @@ module.exports = {
   content: [
     './app/**/{**,.client,.server}/**/*.{js,jsx,ts,tsx}',
     '../../apps/variant/src/**/*.{html,js,ts,tsx,jsx}',
-    '../../components/**/*.{html,js,ts,tsx,jsx}',
+    '../../components/base/**/*.{html,js,ts,tsx,jsx}',
+    '../../components/feature/**/*.{html,js,ts,tsx,jsx}',
+    '../../components/lib/**/*.{html,js,ts,tsx,jsx}',
+    '../../components/stories/**/*.{html,js,ts,tsx,jsx}',
```

Comment:
@francisl Pour etre plus générique

```suggestion
      '../../components/**/*.{html,js,ts,tsx,jsx}',
      '!../../components/node_modules/**/*',
```